node/cn/handler_test.go

```diff
@@ -249,7 +237,147 @@ func TestBroadcastCNTx_CN_Exists(t *testing.T) {
 	pnPeer.EXPECT().AsyncSendTransactions(gomock.Any()).Times(0)
 	enPeer.EXPECT().AsyncSendTransactions(gomock.Any()).Times(0)
 
-	pm.broadcastCNTx(txs)
+	pm.BroadcastTxs(txs)
+}
+
+func TestBroadcastTxsFromPN_PN_NotExists(t *testing.T) {
+	pm := &ProtocolManager{}
+	pm.nodetype = node.PROXYNODE
+	mockCtrl := gomock.NewController(t)
+	defer mockCtrl.Finish()
+
+	peers := newPeerSet()
+	pm.peers = peers
+	cnPeer, pnPeer, enPeer := createAndRegisterPeers(mockCtrl, peers)
+
+	cnPeer.EXPECT().KnowsTx(tx.Hash()).Return(false).Times(1)
+
+	cnPeer.EXPECT().ConnType().Return(p2p.ConnType(node.CONSENSUSNODE)).Times(1)
+	pnPeer.EXPECT().ConnType().Return(p2p.ConnType(node.PROXYNODE)).Times(1)
+	enPeer.EXPECT().ConnType().Return(p2p.ConnType(node.ENDPOINTNODE)).Times(1)
+
+	pnPeer.EXPECT().KnowsTx(tx.Hash()).Return(true).Times(1)
+
+	cnPeer.EXPECT().SendTransactions(gomock.Any()).Times(1)
```

Comment:
If possible, checking SendTransactions's argument is equal with `txs` is better.
Is it possible? If not, just resolve this. :)

Reply:
I changed `gomock.Any()` to `gomock.Eq()`, however, `TestBroadcastTxsFromCN_CN_Exists` fails. I've investigated the issue, but failed to find the reason. Other than that, all `gomock.Any()` are changed to `gomock.Eq()`.

Reply:
Fixed! Thanks!